android/app/src/main/res/values-ro/strings.xml

```diff
@@ -1691,17 +1691,17 @@
 	<string name="type.shop.hifi">Audio HiFi</string>
 	<string name="type.shop.houseware">Magazin de articole de uz casnic</string>
 	<string name="type.shop.jewelry">Bijutier</string>
-	<string name="type.shop.kiosk">Magazin</string>
+	<string name="type.shop.kiosk">Chioșc</string>
 	<string name="type.shop.kitchen">Magazin de bucatarie</string>
 	<string name="type.shop.laundry">Spălătorie</string>
 	<string name="type.shop.mall">Mall</string>
 	<string name="type.shop.massage">Salon de masaj</string>
 	<string name="type.shop.mobile_phone">Magazin de telefoane mobile</string>
-	<string name="type.shop.money_lender">Magazin</string>
+	<string name="type.shop.money_lender">Creditor de bani</string>
```

Comment:
This doesn't sound very idiomatic, but on the other hand it's hard to think of a better alternative. Google Maps calls this category "Agenție de facilitări credite".

Reply:
Other options: "IFN" (non-bank financial institution), or simply "Credite".

Reply:
Thanks. Replaced `Creditor de bani` with `Credite`